src/common/serviceUtils.js

```diff
@@ -87,25 +86,32 @@ function handleApiMessages(messages) {
 export function handleRequestError(error) {
   // Validation errors
   if (error.response && error.response.data.field_errors) {
-    logInfo('Field Errors', error.response.data.field_errors);
+    logInfo('API Field Errors', { fieldErrors: error.response.data.field_errors });
     handleFieldErrors(error.response.data.field_errors);
   }
 
   // API errors
   if (error.response && error.response.data.errors !== undefined) {
-    logInfo('API Errors', error.response.data.errors);
+    logInfo('API Errors', { errors: error.response.data.errors });
     handleApiErrors(error.response.data.errors);
   }
 
   // API messages
   if (error.response && error.response.data.messages !== undefined) {
-    logInfo('API Messages', error.response.data.messages);
+    logInfo('API Messages', { messages: error.response.data.messages });
```

Comment:
Left the message generic and added the custom metric name.  It's pretty effective to filter by appName in my limited experience.

Reply:
I'd really like this message to be more clear, even if it doesn't include the app name.  Something like:
```
API error with expected user messages. See messages metric for details.
```
I'm writing this from the experience of looking at "API Messages" in insights and having no clue what it meant.  I don't see why we wouldn't want the message to make things more immediately clear.